go/service/account.go

```diff
@@ -45,9 +45,14 @@ func (h *AccountHandler) PassphrasePrompt(_ context.Context, sessionID int) erro
 		WindowTitle: "Passphrase needed",
 		Prompt:      "Please enter the passphrase to unlock your secret key",
 		Features: keybase1.GUIEntryFeatures{
-			SecretStorage: keybase1.SecretStorageFeature{
-				Allow: true,
-				Label: "Save in Keychain",
+			StoreSecret: keybase1.Feature{
+				Allow:   true,
+				Respond: true,
```

Comment:
Respond means we want the value back. We want storeSecret but don't care about showTyping